draft-ietf-mls-protocol.md

```diff
@@ -1049,8 +641,7 @@ A              B     ...      Z          Directory        Channel
 |              |              |              |              |
 ~~~
 {: #update-flow title="Client B proposes to update its key, and client A commits the
-proposal.  As a result, the keys for both B and A updated, so the group has
-post-compromise security with respect to both of them."}
```

Comment:
"As a result" was supposed to be moved into the body of the text I believe

Reply:
This got reworded as it moved.